subjects/evenlength/README.md

```diff
@@ -0,0 +1,35 @@
+## evenlength
+
+### Instructions
+
+Write a function that receives a slice of strings and returns a new slice with the strings in which the length is even.
```

Comment:
explain what happens when its an empty slice or when there is none even strings

Reply:
Thanks. Done.